projects/transformers/experiments/ablations.py

```diff
@@ -291,6 +290,42 @@ def pct_start_hp_space(trial):
     model_name_or_path="/mnt/efs/results/pretrained-models/transformers-local/bert_sparse_80%_kd_onecycle_lr_100k",  # noqa: E501
 )
 
+#############
+# Deepspeed #
+#############
```

Comment:
Nit pick: For consistency with the other files, could you format the section header by
```
# ------------
# Deepspeed
# ------------
```